pkg/apis/core/types.go

```diff
@@ -5070,6 +5098,11 @@ type SecurityContext struct {
 	// readonly paths and masked paths.
 	// +optional
 	ProcMount *ProcMountType
+	// The seccomp options to use by this container. If seccomp options are
+	// provided at both the pod & container level, the container options
+	// override the pod options.
+	// +optional
+	SeccompProfile *SeccompProfile
```

Comment:
Maybe you know this out of your mind, so allow me one question:

Does it work if the sandbox profile has higher restrictions than the container? For example:
1. Pod sandbox runs on a seccomp profile which forbids `mkdir(2)`
2. Container now runs on a different profile allowing `mkdir(2)`
3. Executing `mkdir` will work inside the container

Reply:
Verified that it works as intended by my own :heavy_check_mark: